torch/distributed/_tensor/examples/comm_mode_features_example.py

```diff
@@ -581,7 +581,9 @@ def example_transformer_operation_tracing(
 
         # print the operation level collective tracing information
         print(comm_mode.generate_comm_debug_tracing_table(noise_level=2))
-        comm_mode.log_comm_debug_tracing_table_to_file(noise_level=2)
+        comm_mode.log_comm_debug_tracing_table_to_file(
```

Comment:
Another feature request, recently I tried our new comm mode features out, and I found that the current compute operation recording only happen if we have a module, I wonder if the compute operation tracing could happen even when we don't have a module? and in this case we just don't need to print out the module hierarchy info, just compute and communication ops to the global table

Reply:
Addressed in the next PR